src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,28 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String print(int size) {
+		if (size <= 0 || size % 2 == 0) {
 			return null;
 		}
-		StringBuilder builder = new StringBuilder(new String(make(n, n)));
-		for (int i = n - 2; i > 0; i -= 2) {
-			char[] chars = make(n, i);
-			builder.insert(0, chars);
-			builder.append(chars);
+		StringBuilder diamond = new StringBuilder(new String(createLine(size, size)));
+		for (int numberOfStars = size - 2; numberOfStars > 0; numberOfStars -= 2) {
+			char[] line = createLine(size, numberOfStars);
+			diamond.insert(0, line);
```

Comment:
Just glancing at the code, it's not obvious to me what these do.
Could it be:
insertLineAbove()
insertLineBelow()
?